### ISSUE TITLE
[monkeys-audio] build failure

### ISSUE THREAD

**Issue by longhuan2018** [microsoft/vcpkg#10512]
```
-- Downloading https://monkeysaudio.com/files/MAC_SDK_514.zip...
-- Downloading https://monkeysaudio.com/files/MAC_SDK_514.zip... Failed. Status: 22;"HTTP response code said error"
CMake Error at scripts/cmake/vcpkg_download_distfile.cmake:175 (message):
      
      Failed to download file.
      If you use a proxy, please set the HTTPS_PROXY and HTTP_PROXY environment
      variables to "https://user:password@your-proxy-ip-address:port/".
      Otherwise, please submit an issue at https://github.com/Microsoft/vcpkg/issues

Call Stack (most recent call first):
  ports/monkeys-audio/portfile.cmake:7 (vcpkg_download_distfile)
  scripts/ports.cmake:90 (include)
```
I did not find the file MAC_SDK_514.zip in the URL https://monkeysaudio.com/files/

**Comment by longhuan2018** [comment]
This problem was discovered when I submitted a new PR #10511. I don't know why the PR will compile this port.

**Comment by NancyLi1013** [comment]
Hi @longhuan2018,

Thanks for posting this issue.
Since this archive could not be found here https://monkeysaudio.com/files/MAC_SDK_514.zip. 
It seems that this file has been removed now.
We have updated it to the latest version included in PR #10508 to fix this port.

**Comment by NancyLi1013** [comment]
Since the PR has been merged. Please try to update vcpkg and rebuild this port.
